app/jobs/iiif_manifest_cache_prewarm_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class IiifManifestCachePrewarmJob < Hyrax::ApplicationJob
+  ##
+  # @param work [ActiveFedora::Base]
+  def perform(work)
+    manifest_builder.manifest_for(presenter: Hyrax::IiifManifestPresenter.new(work))
```

Comment:
This is splitting hairs, a personal preference of mine is to instantiate the object separate from the passing of that object as a parameter. This is a bit of the command/query separation principle.

I've also found this helps in inserting a debugger if necessary.

```ruby
def perform(work)
  presenter = Hyrax::IiifManifestPresenter.new(work)
  manifest_builder.manifest_for(presenter: presenter)
end
```